atm.js

```diff
@@ -1,3 +1,122 @@
+    console.log(checkingInput.html());
+  var checkingDeposit = $(".button input.deposit");
+    console.log(checkingDeposit.html());
+  var checkingWithdraw = $(".button input.withdraw");
+    console.log(checkingWithdraw.html());
+
+  //Users of the ATM object with a Savings Account
+  //Users of the ATM object can deposit money into bank account
+  //Set the value of 0
+
+  var savingsAccount = 0;
+  var savingsBalance = $("#savings div.balance");
+    console.log(savingsBalance.html());
+  var savingsInput = $(".input input.input");
+    console.log(savingsInput.html());
+  var savingsDeposit = $(".button input.deposit");
+    console.log(savingsDeposit.html());
+  var savingsWithdraw = $(".button input.deposit");
+    console.log(savingsWithdraw.html());
+
+
+  //Create an event that listens for click on deposit button from user when a deposit is made
+  //When deposit button is clicked it will console.log('hello')
+  //This works for both accounts
+
+
+ $(".deposit").on("click", function() {
+      console.log("hello");
+
+ });
+
+ //When deposit button is clicked it will return the userinput
+ //When userinput is returned it will console.log('get money')
+ //When the deposit button is clicked, the balance container will update (in add. to current) with userinput
+
+
+$("div#checking > .deposit").on("click", sumTotal);
+function sumTotal() {
+  var checkingBalance = parseInt($("#checking > .input").val());
+    if($(this).val()==$(this).attr('.deposit'))
+         $(this).val().split();
+   console.log(checkingBalance);
+   console.log("in sumTotal");
+}
+
+
+
+//Users of the ATM object can withdraw money from bank account
+//Create an event that listens for click on withdraw button from user when a withdraw is made
+//When withdraw button is clicked it will console.log('bye')
+//This does it for both
+
+$(".withdraw").on("click", function() {
+     console.log("bye");
+
+});
+
+//When withdrawn button is clicked it will return the userinput
+//When userinput is returned it will console.log('ballin')
+//When the withdraw button is clicked, the balance container will update (in add. to current) with userinput
+
+
+("#checking > .withdraw").on("click", sumTotal);
+
+function sumLoss() {
+  var checkingBalance = parseInt($("#checking > .input").val());
+    if($(this).val()==$(this).attr('.withdraw'))
+         $(this).val().split();
+   console.log(checkingBalance);
+   console.log("in sumLoss");
+}
+
+
+//----------------------------------------------------
+
+//When withdrawn button is clicked it will return the userinput
+//When userinput is returned it will console.log('ballin')
+//When the withdraw button is clicked, the balance container will update (in add. to current) with userinput
+
+$("div#savings > .deposit").on("click", sumTotal);
+function sumSavingsTotal() {
+  var checkingBalance = parseInt($("#savings > .input").val());
+    if($(this).val()==$(this).attr('.deposit'))
+         $(this).val().split();
```

Comment:
I'm not sure what you're trying to do here, but $(this).attr('class') is what would give the account type in this example. there isn't an html attribute called `.deposit`.

`.split("")` creates an array out of a string. I think you were trying to determine whether or not you'd be able to withdraw from your account based on the balance. 

Guessing just plain doesn't work in programming. You need to be very specific and intentional about what you are doing. It's part of the difficulty of the work!

This is one reason why it makes sense to pseudo code, but also declaring variables with meaningful names like `amountRequested` and `checkingBalance`. Pseudocode or identify the information you need, devise names for your variables that reflect what information they hold, and then define them with the appropriate data.

Easier said than done, but what I want to stress is that we, as programmers, are forced to repeatedly break down problems with accuracy and specificity. 

I'd like you to pseudocode when you have a larger assignment or lab, because I think it would help you simplify/clarify your thinking about a certain problem.